packages/backend/src/modules/update-monitor/utils/findUnknownEntries.test.ts

```diff
@@ -1,12 +1,18 @@
 import type { ConfigReader, EntryParameters } from '@l2beat/discovery'
-import { EthereumAddress } from '@l2beat/shared-pure'
+import { ChainSpecificAddress, EthereumAddress } from '@l2beat/shared-pure'
 import { expect, mockFn, mockObject } from 'earl'
 
 import { findUnknownEntries } from './findUnknownEntries'
 
-const A = { address: EthereumAddress.random() }
-const B = { address: EthereumAddress.random() }
-const C = { address: EthereumAddress.random() }
+const A = {
+  address: ChainSpecificAddress.from('eth', EthereumAddress.random()),
```

Comment:
Just a thought - we use `ChainSpecificAddress.from('eth, addr)` so much, that maybe it's worth adding `ChainSpecificAddress.eth(addr)` helper. But no strong opinion on this. It just would be shorter.

Btw, I think in the future we should rename `ChainSpecificAddress` to just `Address` :)